src/Microsoft.IdentityModel.Protocols/Configuration/ConfigurationManager.cs

```diff
@@ -322,6 +340,19 @@ public override void RequestRefresh()
             }
         }
 
+        internal void TrackGetConfiguration(long methodStartTicks, string refreshReason, string operationStatus, string exceptionType)
```

Comment:
I'd prefer this to have two methods, one with exceptionType, one without. It is confusing above to pass string.empty most of the time.